Add web login APIs (#87)
jfrog/jfrog-client-js: README.md

    console.error(error);
  });
```

> 

Reviewer: Please run reformat on the README. I found some missing line breaks.

### Platform

#### Web Login

> #### Register For Web Login

Reviewer: You can't make the title its parent to have the same number of #
```suggestion
##### Register For Web Login
```

> #### Register For Web Login

Reviewer: Let's add all titles to the table of contents

> To register for web login, use the following code:

Reviewer: This comment does not add more info than the title.
```suggestion
```

```javascript
const sessionId = XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX // UUID
jfrogClient
  .platform()
  .WebLogin()
  .registerSessionId(sessionId)
  .then((result) => {
    ...
  })
  .catch((error) => {
    ...
  });
```

> #### Get Access Token From Web Login

Reviewer: You can't make the title its parent to have the same number of #
```suggestion
##### Get Access Token From Web Login
```

Please also add the title to the table of contents

> To retrieve an access token from web login, use the following code:

Reviewer: This comment does not add more info than the title.
```suggestion
```

```javascript
const sessionId = XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX // UUID
jfrogClient
  .platform()
  .WebLogin()
  .waitForToken(sessionId)
  .then((result) => {
    ...
  })
  .catch((error) => {
    ...
  });
```

> Please note that you need to replace 'XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX' with the actual session ID you want to use.

Reviewer: ```suggestion
Please note that you need to replace 'XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX' with the actual session ID that you've generated for `registerSessionId`.
```
(and registerSessionId is a link)

    console.error(error);
  });
```

### Platform

#### Web Login

##### Register For Web Login

```javascript
const sessionId = XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX // UUID
jfrogClient
  .platform()

>   .WebLogin()

Reviewer: ```suggestion
  .webLogin()
```

  .registerSessionId(sessionId)
  .then((result) => {
    ...
  })
  .catch((error) => {
    ...
  });
```

##### Get Access Token From Web Login

```javascript
const sessionId = XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX // UUID
jfrogClient
  .platform()

>   .WebLogin()

Reviewer: ```suggestion
  .webLogin()
```
